### PR TITLE
hv: hv_main: adjust vm exit trace position

### PR DESCRIPTION
**Pull request by zhipenggong**
When we use script to analyze the latency between vm_exit and vm_entry,
it does not include the duration of vm_exit handler.

The patch moves the vm_exit trace before vmexit_handler.

Tracked-On: #2341
Signed-off-by: Zhipeng Gong <zhipeng.gong@intel.com>
Reviewed-by: Yan, Like <like.yan@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
2342<br>No new violations to the coding guideline detected.

**Comment by wenlingz** [comment]
@dongyaozu please help approve it,thanks

File: hypervisor/common/hv_main.c [hunk 1]
```diff
@@ -74,18 +74,18 @@ void vcpu_thread(struct sched_object *obj)
 		profiling_pre_vmexit_handler(vcpu);
 
 		CPU_IRQ_ENABLE();
+		basic_exit_reason = vcpu->arch.exit_reason & 0xFFFFU;
+		TRACE_2L(TRACE_VM_EXIT, basic_exit_reason, vcpu_get_rip(vcpu));
+
 		/* Dispatch handler */
```
Comment (dongyaozu) [inline hypervisor/common/hv_main.c]: This is still too later..

**Comment by HuiHuangShi** [comment]
2342<br>No new violations to the coding guideline detected.

File: hypervisor/common/hv_main.c (hunk 1 above)
Comment (zhipenggong) [reply hypervisor/common/hv_main.c]: Move the trace right after run_vcpu

**Pull request merged**